velox/exec/Spill.cpp

```diff
@@ -117,6 +84,45 @@ SpillState::SpillState(
       pool_(pool),
       stats_(stats) {}
 
+std::vector<std::pair<column_index_t, CompareFlags>>
+SpillState::makeSortingKeys(
+    size_t numKeys,
```

Comment:
@duanmeng : If numKeys is always compareFlags.size, then we can avoid that parameter entirely. Is there something else happening here ?

Reply:
I think compare flags could be empty as defaults. See code below @duanmeng @tanjialiang do we have such use cases? Thanks!